docs/configuration/defaults.md

```diff
@@ -0,0 +1,221 @@
+The top-level `defaults` section specifies type-global:
+
+1. Preprocessing
+2. Encoder
+3. Decoder
+4. Loss
+
+Any configurations set in the `defaults` section apply to all features of that particular feature type.
+Any feature-specific `preprocessing` and `encoder` configurations will be applied to all input features
+of that feature type, while `decoder` and `loss` configurations will be applied to all output features
+of that feature type.
+
+These parameters can be set for specific features through the [input feature configuration](../features/input_features#preprocessing) or [output feature configuration](../features/output_features#decoders).
+
+## Defining Defaults
+
+The defaults section of Ludwig has the following structure:
+
+```yaml
+defaults:
+    <feature_type_1>:
+        preprocessing:
+            parameter_1: value
+            ...
+        encoder:
+            parameter_1: value
+            ...
+        decoder:
+            parameter_1: value
+            ...
+        loss:
+            parameter_1: value
+            ...
+    <feature_type_2>:
+        preprocessing:
+            parameter_1: value
+            ...
+        encoder:
+            parameter_1: value
+            ...
+        decoder:
+            parameter_1: value
+            ...
+        loss:
+            parameter_1: value
+            ...
+    ...
+```
+
+Each of the sections `preprocessing`, `encoder`, `decoder` and `loss` are optional so you can
+define one or more as you need.
+
+### Type-Global Preprocessing
+
+Specify preprocessing policies that apply globally across all input features of a
+certain data type. For example:
+
+```yaml
+defaults:
+    category:
+        preprocessing:
+            missing_value_strategy: fill_with_const
+            fill_value: <UNK>
+```
+
+The preprocessing parameters that each data type accepts can be found in [datatype-specific documentation](../features/supported_data_types).
+
+Note that different features with the same datatype may require different preprocessing. Type-global preprocessing works
+in tandem with feature-specific preprocessing configuration parameters, which override the global settings.
```

Comment:
I think we may want to reword this and put it completely in bold so it's very clear that feature-specific configurations override defaults. I think you could even just change it to say something like: "Note: feature-specific configurations will override parameters set in the default section" or something along those lines. My reasoning is that this is a very important detail and having the extra words here increases the chances that the user doesn't completely understand it. WDYT?

Reply:
@connor-mccorm Agreed! I've added this more explicitly with an example at the top - good call out!